libs/gpg45-evaluator/src/main/java/uk/gov/di/ipv/core/library/gpg45/Gpg45ProfileEvaluator.java

```diff
@@ -54,152 +44,72 @@ public Optional<Gpg45Profile> getFirstMatchingProfile(
                 .findFirst();
     }
 
-    public Gpg45Scores buildScore(List<VerifiableCredential> vcs)
-            throws UnknownEvidenceTypeException, CredentialParseException {
-        var evidenceMap = parseGpg45ScoresFromCredentials(vcs);
-        processEvidenceItems(evidenceMap, CredentialEvidenceItem.EvidenceType.DCMAW);
-        processEvidenceItems(evidenceMap, CredentialEvidenceItem.EvidenceType.F2F);
-        processFraudWithActivityItems(evidenceMap);
+    public Gpg45Scores buildScore(List<VerifiableCredential> vcs) {
+
+        var identityChecks =
+                vcs.stream()
+                        .filter(vc -> vc.getCredential() instanceof IdentityCheckCredential)
+                        .flatMap(
+                                vc ->
+                                        ((IdentityCheckCredential) vc.getCredential())
+                                                .getEvidence().stream())
```

Comment:
Do we need a null-check on evidence here? I don't think it should ever be null for an `IdentityCheckCredential`...

Reply:
According to [this](https://vocab.account.gov.uk/v1/json-schemas/IdentityCheckCredential.json), evidence is required so I don't think we need to do a null check here